### ISSUE TITLE
Suggestion: Connected Plane

### ISSUE THREAD

**Issue by CptPICHU** [ChiselsAndBits/Chisels-and-Bits#767]
As already requested on Discord, i'd suggest a Connected Plane - chisel mode.
How exactly it works:
-The main method is just the "Plane mode".
But it also applied to everything that is connected to the clicked part using PLANE mode.

![picture](https://user-images.githubusercontent.com/71035803/137645161-559badcf-e5a4-4649-8149-2e36f63b010d.png)
When you keep chopping away the upper part, at the end this will be left:
![picture2](https://user-images.githubusercontent.com/71035803/137645191-1b62cac2-fd18-4cf3-8065-581e7a622370.png)

Basically:
Plane chisel mode. But everything that is NOT directly connected will not be chopped away.
The best way to test this i using the pre-rebuild c&b, this has this exact feature.

**Comment by creamIcec** [comment]
Same suggestion. (Actually this is originally suggested by me) 
I hope If any part of a block which is not connected to any other part of same material can be removed using a particular chisel mode. This will bring convenience to the process of any chiseling.
